docs/tls.md

```diff
@@ -76,10 +80,34 @@ spec:
     name: selfsigning-issuer
 ```
 
-Finally configure MinIO to use the newly created TLS certificate:
+Finally, configure MinIO to use the newly created TLS certificate:
 
 ```yaml
   externalCertSecret:
     - name: tls-minio
       type: kubernetes.io/tls
 ```
+
+---
+
+## Rotate MinIO Operator TLS certificates
+
+In case the operator TLS certificate expired you can just delete the `operator-tls` secret in the `minio-operator` namespace and
+operator will generate a new one automatically, additionally it will copy the `public.crt` portion of the certificate into
+each tenant namespace
+
+---
+
+## Using your own certificate for MinIO Operator
+
+If you wish to use your own certificate for `MinIO Operator` instead of using the auto generated one, just delete or replace the content of the
+`operator-tls` secret in the `minio-operator` namespace, `operator-tls` must contain the `public.crt` and `private.key` keys.
+
+---
+
+## Using your own CA Certificate for MinIO Operator
+
+If your `MinIO` tenants are using `custom certificates` or certificates generated by your own internal `certificate authority` (ie: `cert-manager`).
+`MinIO Operator` needs to trust the `TLS` connections in order to talk to the `MinIO tenants`, for that you need to create a new secret in the
+`minio-operator` namespace named `operator-ca-tls`, inside this secret create a new key `ca.crt` that will include the public certificate
```

Comment:
Can this have multiple ca.crt if for some reason users have multiple CA for tenants?